source/_integrations/prusalink.markdown

```diff
@@ -18,16 +18,17 @@ ha_platforms:
 ha_dhcp: true
 ---
 
-The PrusaLink integration allows you to monitor your [Prusa 3D printer](https://www.prusa3d.com) and its progress with your Home Assistant installation. This integration works with Prusa MINI/MINI+, Prusa MK3.9/MK4 and Prusa XL. It does not work with the older Raspberry Pi based Prusa Link printers.
+The PrusaLink integration allows you to monitor your [Prusa 3D printer](https://www.prusa3d.com) and its progress with your Home Assistant installation. This integration works with Prusa MINI/MINI+, Prusa MK3.9/MK4, Prusa XL and with the older Raspberry Pi based Prusa MK2.5/MK3.
 
-This integration has been updated to utilize the latest v1 API endpoints, which require firmware version 4.7.0 or later. If you own a Prusa MINI/MINI+, please make sure your printer is running firmware 5.1.0 or a more recent version. Firmware versions 4.7.x and 5.0.x are not available for this model.
+This integration has been updated to utilize the latest v1 API endpoints, which require firmware version 4.7.0 or later. If you own a Prusa MINI/MINI+, please make sure your printer is running firmware 5.1.0 or a more recent version. Firmware versions 4.7.x and 5.0.x are not available for this model. For Prusa MK2.5/MK3, this integration requires PrusaLink version 0.7.2 or later.
 
 Firmware update guides can be found here:
 
  - [Prusa MINI/MINI+](https://help.prusa3d.com/article/firmware-updating-mini-mini_124784)
  - [Prusa MK3.9/MK4/XL](https://help.prusa3d.com/article/how-to-update-firmware-mk4-xl_453086)
+ - [Prusa MK2.5/MK3](https://help.prusa3d.com/guide/how-to-update-prusalink-mk2-5-s-mk3-s-_650837)
 
-To obtain the hostname, username, and password:
+To obtain the hostname, username, and password on Prusa MINI/MINI+/MK3.9/MK4/XL:
```

Comment:
Could you document on how to obtain the API token on MK2.5/3?

Reply:
I didn't document it because there's no good way to retrieve the credentials from MK2.5/MK3. (You enter the credentials when you first set up PrusaLink, after that you have to remember them or use a password manager etc.)

I'll try to mention this in the docs somehow.

Reply:
Updated: https://deploy-preview-32031--home-assistant-docs.netlify.app/integrations/prusalink/#obtaining-hostname-username-and-password